File: C/reverse.c
//REVERSE OF AN ARRAY
#include<stdio.h>
void main()
{
	int i,a[50],n,temp;
	printf("Enter size of array: ");
	scanf("%d",&n);
	printf("Enter elements of array:\n");
	for(i=0;i<n;i++)
		scanf("%d",&a[i]);
	for(i=0;i<n/2;i++)
	{
		temp=a[i];
		a[i]=a[n-1-i];
		a[n-1-i]=temp;
	}
	printf("Reversed array is:\n");
	for(i=0;i<n;i++)
		printf("%d ",a[i]);
	printf("\n");
}

File: Java/Reverse.java

//Write a program to display the reverse of entered number
import java.io.*;
class Reverse
{
	public static void main(String args[]) throws IOException
	{
		int num,a;
		BufferedReader br=new BufferedReader(new InputStreamReader(System.in));
		System.out.println("Enter the number to be reversed:");
		num=Integer.parseInt(br.readLine());
		System.out.print("The Reversed Number is ");
		while(num>0)
		{
			a=num%10;
			System.out.print(a);
			num=num/10;
		}
		System.out.println();
	}
}

/*
Output:
Enter the number to be reversed:
3428
The Reversed Number is 8243
*/
